apps/developer-hub/src/app/page.tsx

```diff
@@ -0,0 +1,15 @@
+
+import { Homepage } from "../components/Homepage/index.js";
+import { HomepageHero } from "../components/HomepageHero/index.js";
+
+const Page = () => {
+  return (
+    <Homepage>
+      <HomepageHero />
+      
+      {/* Other sections will be added here */}
+    </Homepage>
+  );
+};
+
```

Comment:
Definitely not a requirement and feel free to disregard, but I've found that I prefer keeping everything in the nextjs `app` directory as just re-exports of components from the `components/` directory, along with metadata and other nextjs page config.  That gives me a very clean separation of concerns -- the `app` directory is concerned with configuring the urls, nextjs caching, and metadata, but all UI lives in `components`.

Up to you if you choose to follow that as well but I've found I like it more than having some UI in `app` and others in other places in the repo, it's a bit more predictable and simpler.